net/tailscale/Makefile

```diff
@@ -61,6 +61,28 @@ endef
 
 Package/tailscaled/description:=$(Package/tailscale/description)
 
+GO_IPTABLES_VERSION:=0.6.0
+GO_IPTABLES_FILE:=$(PKG_VERSION)-go-iptables-$(GO_IPTABLES_VERSION).tar.gz
```

Comment:
```suggestion
GO_IPTABLES_FILE:=go-iptables-$(GO_IPTABLES_VERSION).tar.gz
```

this source file is generic anyway... or use `$(PKG_NAME)-go...` instead?

Reply:
I'll fix that.. that's a typo, it should had been pkg_name, not version..